### PR TITLE
New Layout for edit profile

### PR DESCRIPTION
**Pull request by sreeramgoalla**
edit_profile

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -51,6 +51,12 @@ def destroy
     end
   end
 
+  def check_for_cancel
```
Comment (lemattma) [inline app/controllers/recipes_controller.rb]: @sreeramgoalla what's this for?

File: app/views/devise/registrations/edit.html.haml [hunk 2]
```diff
@@ -1,24 +1,53 @@
...
+                    %p.heading
+                      Profile Picture
+                      .text-right
+                        = link_to 'Upload', '#', class: 'btn btn-success'
 
-= link_to 'Back', :back
+    .row
+      .col-md-3
+        %h4 Delete Your Account ?
+        %p= link_to 'Delete Account', registration_path(resource_name), data: { confirm: "Are you sure?" }, class: 'btn btn-danger btn-lg' ,method: :delete
```
Comment (lemattma) [inline app/views/devise/registrations/edit.html.haml]: Please change `,method: :delete` to `, method: :delete` :)

**Comment by sreeramgoalla** [comment]
@lemattma Any more updates/comments on this ?

**Comment by lemattma** [comment]
Looks good, you'll need to rebase and fix conflicts.

**Comment by sreeramgoalla** [comment]
@lemattma Sure

**Pull request closed without merging**